src/main/scala/rocket/PTW.scala

```diff
@@ -74,7 +79,8 @@ class DatapathPTWIO(implicit p: Parameters) extends CoreBundle()(p)
   val customCSRs = coreParams.customCSRs.asInput
   val clock_enabled = Bool(OUTPUT)
 }
-
+/** Page table entry
+ *  @see RV-priv spec 4.3.1 for pgae table entry format*/
```

Comment:
Glad you have found the spec :)
```suggestion
/** Page table entry
  * @see RV-priv spec 4.3.1 for pgae table entry format*/
```